projects/ad9172/src/README

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
eventually this will have to be replaced with a makefile, but that we will figure out later, just something to be aware of.